modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -1223,6 +1222,72 @@ public void testAddObjectActionWithConditionExpression() throws Exception {
 		_objectActionLocalService.deleteObjectAction(objectAction4);
 	}
 
+	@Test
+	public void testAddObjectActionWithLabelMap() throws Exception {
+		Locale locale = LocaleUtil.fromLanguageId(
+			_objectDefinition.getDefaultLanguageId());
+
+		String objectActionName = RandomTestUtil.randomString();
```

Comment:
@pedro-oliveira446 use static names, `"objectAction1Name"`, just for read purpose, static values are better in this context.

Thanks!